server/data/data-jmap-cassandra/src/test/java/org/apache/james/jmap/cassandra/access/CassandraAccessTokenRepositoryTest.java

```diff
@@ -31,7 +32,7 @@
 @ContractImpl(CassandraAccessTokenRepository.class)
 public class CassandraAccessTokenRepositoryTest {
 
-    private IProducer<CassandraAccessTokenRepository> producer = new IProducer<CassandraAccessTokenRepository>() {
+    private IProducer<AccessTokenRepository> producer = new IProducer<AccessTokenRepository>() {
```

Comment:
sorry but it's more about using the diamond operator : 
`private IProducer<AccessTokenRepository> producer = new IProducer<>() {`